packages/fx-core/templates/core/v3Migration/js.ts.app.yml

```diff
@@ -152,6 +152,8 @@ deploy:
   - uses: prerequisite/install # Install dependencies
     with:      
       dotnet: true
+    writeToEnvironmentFile: # Write the information of installed dependencies into environment file for the specified environment variable(s).
+      dotnetPath: DOTNET_PATH
```

Comment:
Also check whether needs to align with `dotnetPath` in line 166

Reply:
same as above.